apps/files/models.py

```diff
@@ -80,6 +80,8 @@ class File(amo.models.OnChangeMixin, amo.models.ModelBase):
     binary_components = models.BooleanField(default=False, db_index=True)
     # Serial number of the certificate use for the signature.
     cert_serial_num = models.CharField(max_length=255, blank=True)
+    # Is the file signed by Mozilla?
+    is_signed = models.BooleanField(default=False)
```

Comment:
Can there be multiple files per version and some of those not signed? If not, I'm curious why not put this on the `Version` model since that's where the version field is and it has the `.1-signed` in it.